doo-swagger.json

```diff
@@ -5925,6 +6204,300 @@
         "data",
         "_links"
       ]
+    },
+    "WebhooksPostRequest": {
+      "type": "object",
+      "required": [
+        "user",
+        "description",
+        "hooks"
+      ],
+      "properties": {
+        "user": {
```

Comment:
What means user? What value should this property contain?

Reply:
In the documentation I see `name` but not a `user` → https://viovendi.atlassian.net/wiki/spaces/DOO/pages/2298380/As+zapier+I+want+to+get+notified+by+a+webhook+for+new+data+so+that+I+can+stop+polling+for+new+data.